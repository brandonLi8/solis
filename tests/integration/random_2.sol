# Copyright Â© 2022-2023 Brandon Li. All rights reserved.

##
Integration test of a randomly generated program.
NOTE: this file was auto-generated with the `random_program_gen.py` script.
##

467;
let a1: int = -(55);
let b1: bool = (440) >= (43.09509914479138);
let c1: int = 518;
!(false);
!(b1);
-(824);
b1;
-(653);
37.60277548401413;
-(27.271911527379313);
(-((a1) / (41.728947736963576))) < ((-(a1)) * (-((-(a1)) - (a1))));
!(b1);
let d1: int = 283;
13.463945708796428;
let e1: float = -(22.135985671411838);
e1;
(328) - (e1);
(-(505)) % (c1);
b1;
let f1: bool = false;
let g1: float = -(e1);
let h1: float = 26.952816056936317;
let i1: int = (-(290)) % (-(d1));
34.3463100590094;
h1;
let j1: int = -(192);
-(a1);
!(false);
!((-(c1)) > (-(g1)));
-((21.64658058146439) + (g1));
953;
let k1: bool = f1;
!(b1);
-(374);
let l1: bool = !(f1);
c1;
-(h1);
(-((34.562262477510814) / (39.472158256908656))) + (c1);
let m1: float = e1;
-(12.154110653410987);
false;
-(i1);
let n1: int = c1;
false;
let o1: int = -(653);
i1;
e1;
let p1: float = 28.129716144009777;
let q1: int = -(575);
let r1: float = m1;
let s1: float = -(14.239498262054154);
a1;
let t1: int = o1;
let u1: float = 25.532164889541395;
let v1: float = -(20.57770426016929);
922;
(-(355)) * (-(3.670036975163181));
26.93871074928677;
let w1: bool = true;
let x1: bool = true;
let a2: int = -(i1);
-((731) - (32.91452772713274));
86;
-(141);
let b2: bool = !((-(15.848641193711613)) != (-(1.9388284925659132)));
let c2: float = ((-(344)) + (-(3.5313662209746917))) - (-(u1));
-(c1);
-(j1);
k1;
p1;
-(895);
let d2: int = -(43);
u1;
-(33.73251457289953);
-(199);
let e2: int = -(450);
(-(41.98385592500904)) <= (-(c1));
let f2: bool = b1;
a1;
b2;
false;
38.08147828451114;
!(l1);
!(false);
-(406);
let g2: bool = !(f2);
let h2: float = 6.571564626669417;
(10.906882652476408) <= (-((954) - (-(26.85745986778776))));
false;
-(820);
let i2: bool = false;
!(false);
let j2: bool = !(((n1) - ((d2) / (-(836)))) != (-(a2)));
j2;
21.02586355162285;
-(922);
let k2: int = q1;
(false) != (!(i2));
let l2: float = m1;
let m2: float = e1;
(28.344500446010034) - (-(d2));
let n2: float = 8.542801632082721;
886;
let o2: int = a2;
let p2: int = -(d2);
-(285);
-((u1) - (22.792076426943375));
true;
!(false);
(-(a1)) < (-(22.7650160708135));
d2;
619;
t1;
-(r1);
n2;
!(k1);
l2;
l2;
let q2: bool = !((-(p1)) != (-(v1)));
let r2: bool = !((h1) >= (-(e1)));
115;
-(69);
l2;
let s2: float = -(2.9020338731894397);
true;
let t2: int = 113;
let u2: bool = false;
11.490022065261556;
false;
let v2: float = c2;
!(u2);
!(false);
let w2: int = -(589);
if !((-(969)) > (-(r1))) {
  let x2: int = o2;
  !(true);
  !(true)
} else {
  let a3: bool = (false) != (u2);
  38.943626650002464;
  ((187) / (737)) != (-(k2));
  !((-(137)) != (c1))
};
b1;
let b3: bool = !((29.2606494228087) == (-(21.228886249117995)));
let c3: bool = (-(l2)) != (-(g1));
let d3: bool = false;
let e3: int = c1;
!(u2);
false;
let f3: int = -(j1);
let g3: int = t1;
40.2878280489764;
if !((-(12.803841309484412)) >= (-(p2))) {
  -(d1);
  let h3: bool = !(f1);
  -(o2);
  !(true)
} else if !((549) > (-((-(w2)) + (-(542))))) {
  -(43.66767296560696);
  let i3: float = (605) - (-(m2));
  let j3: int = -(w2);
  l1;
  let k3: float = 13.773015962682708;
  -(16.46501423161553);
  !((t2) >= (d2));
  f1
} else if !(false) {
  k2;
  -(50.01776288219059);
  92;
  !(f2)
} else {
  117;
  let l3: bool = !(true);
  e2;
  !(w1);
  -(t2);
  !(true);
  !(f2)
};
let m3: float = -(l2);
46.75954912536154;
let n3: int = 700;
f3 + t1 + k2 + g3 + e2 + d1 + p2 + n3 + o1 + i1 + a2