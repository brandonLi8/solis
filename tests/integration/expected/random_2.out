-4567854283238423382